src/Sulu/Bundle/SecurityBundle/Controller/RoleSettingController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Controller;
+
+use Doctrine\ORM\EntityManagerInterface;
+use FOS\RestBundle\Controller\Annotations\RouteResource;
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use Sulu\Bundle\SecurityBundle\Entity\Role;
+use Sulu\Bundle\SecurityBundle\Entity\RoleSetting;
+use Sulu\Component\Rest\RestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Endpoint for role-settings.
+ *
+ * @RouteResource("setting")
+ */
+class RoleSettingController extends RestController implements ClassResourceInterface
+{
+    /**
+     * Returns value for given role-setting.
+     *
+     * @param int $roleId
+     * @param string $key
+     *
+     * @return Response
+     */
+    public function getAction($roleId, $key)
+    {
+        return $this->handleView(
+            $this->view($this->get('sulu_security.role_setting_repository')->findSettingValue($roleId, $key))
+        );
```

Comment:
Please don't nest all function calls. Writing function calls above each other makes the code easier to adapt later on.